core/scripts/cli/sponsor/redeem.js

```diff
@@ -30,7 +30,8 @@ const redeem = async (web3, artifacts, emp) => {
     name: "numTokens",
     message: "How many tokens to repay, at " + fromWei(collateralPerToken) + " ETH each?",
     validate: value =>
-      (value > 0 && toWei(value) <= walletTokens) || "Number of tokens must be positive and up to your current balance"
+      (value > 0 && toBN(toWei(value)).lte(toBN(walletTokens))) ||
```

Comment:
I ran into errors where certain numbers would not be accepted that were below my balance. I think using BN.js addresses this change. 

For ex. I created a position with 150 collateral and 100 tokens and tried to redeem 20.